tensorboard/webapp/app_routing/route_contexted_reducer_helper.ts

```diff
@@ -185,10 +185,10 @@ export function createRouteContextedState<
 
         if (
           !areSameRouteKindAndExperiments(before, after) &&
-          onRouteIdChanged
+          onRouteKindOrExperimentsChanged
         ) {
-          // areSameRouteKindAndExperimentsperiments have changed. Delegate additional changes to the caller.
-          nextFullState = onRouteIdChanged(nextFullState, after);
+          // Experiments have changed. Delegate additional changes to the caller.
```

Comment:
`Route kinds or experiments have changed`? Or only experiments will be changed?

Reply:
Right, updated.